LayoutTests/platform/wpe/TestExpectations

```diff
@@ -148,6 +144,9 @@ imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-adjustment_clic
 imported/w3c/web-platform-tests/pointerevents/pointerup_after_pointerdown_target_removed.html?touch [ Skip ]
 imported/w3c/web-platform-tests/pointerevents/predicted_events_attributes.html?touch [ Skip ]
 
+# webkit.org/b/274686 -- Crashing under PointerCaptureController::processPendingPointerCapture
+imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_mouse_and_release_and_capture_again.html [ Crash ]
```

Comment:
This is glib and mac mainly because the entire suite is skipped on iOS: https://github.com/WebKit/WebKit/blob/60717e784729ccc4d0239f8027f09ecebf09237d/LayoutTests/platform/ios/TestExpectations#L6584

Reply:
Unfortunately I can't place this expectation in the top-level TestExpectations because the more platform-specific blanket PASS expectation in `LayoutTests/platform/mac/TestExpectations` would override it! (I tried!)

https://github.com/WebKit/WebKit/blob/60717e784729ccc4d0239f8027f09ecebf09237d/LayoutTests/platform/mac/TestExpectations#L93